templates/shared/forms/interactive/esm.html

```diff
@@ -0,0 +1,168 @@
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <h2 class="p-modal__title u-sv1" id="modal-title">Extended Security Maintenance (ESM)</h2>
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">If you would like to learn more about continued security updates for the Ubuntu base OS and critical infrastructure components - Ceph, OpenStack, Kubernetes and more - contact us to discuss a migration path or to get started with ESM for Ubuntu 16.04 or 14.04 LTS.</p>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Where are you running Ubuntu 16.04 or Ubuntu 14.04?</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="workstations">
+            <label for="workstations">Workstations</label>
+            <input type="checkbox" id="physical-servers">
+            <label for="physical-servers">Physical servers</label>
+            <input type="checkbox" id="virtual-machines">
+            <label for="virtual-machines">Virtual machines</label>
+            <input type="checkbox" id="public-cloud">
+            <label for="public-cloud">Public cloud</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="storage">
+            <label for="storage">Storage</label>
+            <input type="checkbox" id="kubernetes">
+            <label for="kubernetes">Kubernetes</label>
+            <input type="checkbox" id="openstack">
+            <label for="openstack">OpenStack</label>
+            <input type="checkbox" id="private-cloud">
+            <label for="private-cloud">Private cloud</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="development-and-ci/cd">
+            <label for="development-and-ci/cd">Development and CI/CD</label>
+            <input type="checkbox" id="production">
+            <label for="production">Production</label>
+            <input type="checkbox" id="mission-critical">
+            <label for="mission-critical">Mission-critical</label>
+          </div>
+        </div>
+      </div>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Tell us more about your estate:</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="less-5-machines">
+            <label for="less-5-machines" >&lt;5 machines</label>
+	          <p class="p-form-help-text" style="margin-top: 0">For a small amount of machines you can use your <a href="/advantage">free personal allowance of Essential</a>.</p>
```

Comment:
@anthonydillon @lechsandecki copy suggestion.
```suggestion
	          <p class="p-form-help-text" style="margin-top: 0">For a small amount of machines, <a href="/advantage">Ubuntu ESM is available for free</a>.</p>
```